src/client/api/container.c

```diff
@@ -508,28 +508,61 @@ daos_cont_list_attr(daos_handle_t coh, char *buf, size_t *size,
 	return dc_task_schedule(task, true);
 }
 
+static int
+free_name(tse_task_t *task, void *args)
+{
+	char *name = *(char **)args;
+	free(name);
```

Comment:
(style)  Missing a blank line after declarations

Reply:
yes, why not !! thx ;-)